tests/spec/test-run-api-service.js

```diff
@@ -503,6 +526,31 @@
                 req.url.should.equal(baseURL + 'myfancyrunid/');
                 req.requestHeaders['X-AutoRestore'].should.equal(true);
             });
+            it('should do an GET with & in variable name', function () {
+                var rs = new RunService({ account: account, project: project });
+                rs.load('myfancyrunid', { include: 'score&' });
+
+                var req = server.requests.pop();
+                req.url.should.equal(baseURL + 'myfancyrunid/?include=score&');
+
+            });
+            it('should do an GET with & in variable names', function () {
+                var rs = new RunService({ account: account, project: project });
+                rs.load('myfancyrunid', { include: ['score&', '&score2'] });
+                var req = server.requests.pop();
+                req.url.should.equal(baseURL + 'myfancyrunid/?include=score&,&score2');
+            });
+
+            it('should do an GET with & in long variable names', function () {
```

Comment:
Similar to above, you can skip this test here, it only matters in the variables api service (unless I'm missing something this does)